nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +255,34 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param receivers The list of receivers.
+   * @param field The class field element.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(List<Element> receivers, Element field) {
+    List<AccessPathElement> elements = new ArrayList<>();
+    elements.add(new AccessPathElement(field));
```

Comment:
This seems weird.  Shouldn't `field` be added to the `elements` list last, after all the relevant Elements from `receivers` have been added?